### PR TITLE
add es6 archi

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Model/OrchestraModel.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        return {}
+    }
+
+    /**
+     * @inheritdoc
+     */
+    sync(method, model, options) {
+        options = options || {};
+        let url = model._getSyncUrl()[method.toLowerCase()];
+        if(typeof url != 'undefined') {
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Model/OrchestraModel.js]: if (

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/TemplateManager.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        this._localStorageKey = 'template-underscore-' + environment;
+        this._loadTemplatesLocalStorage();
+    }
+
+    /**
+     * Clear template in cache
+     */
+    clearCache() {
+        this._templates = {};
+        if(typeof localStorage!='undefined') {
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/TemplateManager.js]: if (

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/TemplateManager.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+        return promise
+    }
+
+    /**
+     * Load template underscore in local storage
+     * @private
+     */
+    _loadTemplatesLocalStorage() {
+        if(typeof localStorage!='undefined') {
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/TemplateManager.js]: if (

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/Form/Model/Form.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+            } else if($(tag).hasClass('alert')) {
+                this.$messages.push($(tag));
+            }
+        });
+        this.method = this.$form.attr('method');
+        this.action = this.$form.attr('data-action');
+    }
+}
+
+export default Form;
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/Form/Model/Form.js]: EOF

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/Form/Model/FormBuilder.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+     * @private
+     */
+    static _successGetForm(data, callbackCreate) {
+        let form = FormBuilder.createForm(data);
+
+        callbackCreate(form);
+    }
+}
+
+export default FormBuilder
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/Form/Model/FormBuilder.js]: EOF

**Comment by itkg-nanne** [comment]
:+1:

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Error/ErrorView.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+     */
+    initialize({error}) {
+        this._error = error;
+    }
+
+    /**
+     * Render node tree
+     */
+    render() {
+        this.$el = $('<p>' + this._error.message + '</p>');
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Error/ErrorView.js]: you don't systematically associate a _tpl to a view ?

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/Form/Model/Form.js [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+                url: this.action,
+                method: this.method,
+                data: this.$form.serialize(),
+                statusCode: statusCode,
+                context: this
+            });
+            promise.done(this._formSuccess);
+            promise.fail(this._formError);
+            promise.always(() => {
+                console.log(this);
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/Form/Model/Form.js]: is this ``console.log`` an artifact from your dev ?

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/Form/Model/Form.js [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+     * @param {string} html
+     * @private
+     */
+    _parseHtml(html) {
+        let $html = $(html);
+        this.$messages = [];
+        $html.each((index, tag) => {
+            if ("FORM" === $(tag).prop("tagName")) {
+                this.$form = $(tag);
+            } else if($(tag).hasClass('alert')) {
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/Form/Model/Form.js]: if ($(tag))

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/Form/View/FormView.js [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+    initialize({form}) {
+        this._form = form;
+    }
+
+    /**
+     * Render a form
+     */
+    render() {
+        console.log('render');
+        console.log(this);
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/Form/View/FormView.js]: I think this should be removed

File: BackofficeBundle/Resources/public/underscore/Node/nodeTreeView._tpl.html [hunk 10]
```diff
@@ -0,0 +1,4 @@
+<p>
+    test
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/underscore/Node/nodeTreeView._tpl.html]: Does this should be removed ?

**Pull request merged**